### ISSUE TITLE
"Recently seen"-indicator does not disappear automatically after 10 minutes

### ISSUE THREAD

**Issue by Raiden-GH** [deltachat/deltachat-desktop#2966]
"Recently seen"-indicator does not disappear automatically after 10 minutes. 
Only after restarting DeltaChat or switching to archived chats and back.

**Comment by Jikstra** [comment]
Thank you :) Here the core ticket: https://github.com/deltachat/deltachat-core-rust/issues/3646

**Comment by Simon-Laux** [comment]
technically this is a duplicate of #2926
